lib/p2p/Peer.ts

```diff
@@ -29,9 +30,9 @@ interface Peer {
   on(event: 'handshake', listener: () => void): this;
   once(event: 'open', listener: () => void): this;
   once(event: 'close', listener: () => void): this;
-  once(event: 'ban', listener: () => void): this;
+  once(event: 'reputationEvent', listener: (event: ReputationEvent) => void): this;
```

Comment:
I think "reputationEvent" is redundant since this is already an event, how about just "reputation" ?

Reply:
Agree.